frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/attached-devices.scss

```diff
@@ -64,7 +64,14 @@
   cursor: pointer;
 }
 
-.ceph-ocs-install__disks-modal {
+.ceph-ocs-install__node-list-btn {
+  padding-top: 0;
+  padding-left: 0;
+  padding-right: 0;
+  cursor: pointer;
```

Comment:
```suggestion
```
If you are making the variant as "link" in Button then you will get the cursor as "pointer". You should not need that to set explicitly.